app/helpers/application_helper.rb

```diff
@@ -172,12 +172,6 @@ def labeled_check_box_tags(name, collection, options = {})
     end.join.html_safe
   end
 
-  def html_hours(text)
-    html_safe_gsub(text,
-                   %r{(\d+)\.(\d+)},
-                   '<span class="hours hours-int">\1</span><span class="hours hours-dec">.\2</span>')
```

Comment:
If `hours-dec` is not used anymore, the associated css defined in `frontend/src/global_styles/content/_tables.sass` should be removed as well. On two places: line 58 and line 147

The intent was to use a smaller size (`0.9rem`) for the decimal part of the hours.
It then looks like this:

![image](https://github.com/user-attachments/assets/b760501c-b8d0-462e-b651-271add3d159d)

Just for the record, the css actually does not work as-is. It needs to be changed to something like:
```css
td
  .hours
    font-weight: var(--base-text-weight-bold)
    &.hours-dec
      font-size: 0.9em
```

Reply:
Good catch. I removed the styles. 👍